src/main/kotlin/snyk/container/ContainerIssues.kt

```diff
@@ -33,6 +33,7 @@
     val severity: String,
     val identifiers: Identifiers? = null,
     val cvssScore: String? = null,
+    var CVSSv3: String? = null,
```

Comment:
old version was better, leading a field/attribute with a capitalized letter is always confusing to me

Reply:
and if it's about automatic conversion - let's use the @SerializedName annotation